plugins/src/webrtcsink/imp.rs

```diff
@@ -1427,6 +1427,10 @@ impl WebRTCSink {
             }
         });
 
+        pipeline.set_state(gst::State::Ready)?;
+
+        element.emit_by_name("new-webrtcbin", &[&peer_id.to_value(), &webrtcbin.to_value()])?;
```

Comment:
The `to_value()` calls are not needed here, and also you probably want to just `unwrap()` here. If this fails then you failed typing the signal name or used the wrong types/number of arguments.